### PR TITLE
Add environment map tutorial

### PR DESCRIPTION
**Pull request by iche033**
# 🎉 New feature

## Summary
Add a new tutorial to show how to set an environment map to a mesh and also how to create a the environment map / cube map texture in gimp

## Test it

Build ign-rendering and the open the doxygen index page: `build/ignition-rendering5/doxygen/html/index.html`

Go to Tutorials and you should see the new `Environment map` tutorial entry.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: tutorials/22_environment_map.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+map texture used by ignition rendering needs to be the form of a cube map.
+
+
+## Example mesh with Environment map
+
+An example of a mesh with an environment map can be found in the `ogre2_demo`,
+which uses the ogre2 render engine.
+
+To see the example mesh, build and run the demo:
+
```
Comment (ahcorde) [inline tutorials/22_environment_map.md:16]:
```suggestion
```

File: tutorials/22_environment_map.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+\page envmap Environment Map
+
+This tutorial shows how to add an environment map to a mesh.
+
+An environment map, also known as reflection map, is an efficient way to
+create reflective surfaces using a precomputed texture. The environment
+map texture used by ignition rendering needs to be the form of a cube map.
+
```
Comment (ahcorde) [inline tutorials/22_environment_map.md:8]:
```suggestion
```

**Review by ahcorde — APPROVED** [review]

File: tutorials/22_environment_map.md (hunk 2 above)
Comment (iche033) [reply tutorials/22_environment_map.md:8]: 939fd4de383f04f8e2712edec9b786acee621e59

File: tutorials/22_environment_map.md (hunk 1 above)
Comment (iche033) [reply tutorials/22_environment_map.md:15]: 939fd4de383f04f8e2712edec9b786acee621e59

**Pull request merged**